package.json

```diff
@@ -5,6 +5,9 @@
     "private": true,
     "workspaces": [
         "packages/*",
+        "packages/cli-templates/authentication-typescript",
+        "packages/cli-templates/blank-typescript",
+        "packages/cli-templates/routing-typescript",
```

Comment:
These should already be covered by the line above I think, unless /* doesn't include further nested folders